programs/protocol_event/src/state/event.rs

```diff
@@ -0,0 +1,36 @@
+use anchor_lang::prelude::*;
+
+#[account]
+pub struct Event {
+    pub authority: Pubkey,
+    pub payer: Pubkey,
+
+    pub category: Category,
+    pub event_group: EventGroup,
+
+    pub active: bool,
+
+    pub slug: String, // event identifier e.g. LAFCvLAG@2021-08-28
+    pub name: String, // for display purposes e.g. Los Angeles Football Club vs. LA Galaxy
+
+    pub participants: Vec<Participant>,
+
+    pub expected_start_timestamp: i64,
```

Comment:
Where did we land on `expected_end_timestamp` ? That would be used for figuring out the initial lock time of an inPlay market. Or do we think an operator should calculate that from the expected start timestamp?

Reply:
Operators prerogative based on expected start time ✅